documentation/peer-testing-2/peer-testing-report.md

```diff
@@ -31,9 +31,13 @@ In total, **\_** individuals took part in our peer testing sessions. The members
 **To be completed after the peer testing sessions**
 
 #### High
-
+- Server sometimes accidentally shut down for unexpected reason.
+- Need an extra tab for mentors to view the history of their declined mentee.
```

Comment:
That's an interesting point. The client hasn't mentioned that as being an important aspect for the system, so maybe we could put that as a low priority, which could be a bonus feature for once we finalize the higher priority functionalities.

Reply:
Agree! Thank you for reviewing!